activemq-broker/src/main/java/org/apache/activemq/broker/scheduler/SchedulerBroker.java

```diff
@@ -54,7 +54,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class SchedulerBroker extends BrokerFilter implements JobListener {
+public class SchedulerBroker extends MutableBrokerFilter implements JobListener {
```

Comment:
Wont this be overhead now for use cases not needing this? due to the sync in mutable thats needed.

Reply:
I wasn't able to find any information about possible `AtomicReference` overhead. I believe if there is some, it should be almost not noticeable